src/lib/base.jl

```diff
@@ -104,3 +104,14 @@ end
 # Make @sync work
 # Safe as long as other kinds of mutation are disallowed
 @adjoint push!(refs::Vector{Any}, t::Task) = push!(refs, t), _ -> nothing
+
+# named tuple
+@adjoint function pairs(t::NamedTuple{N}) where N
+    fun = function pairs_namedtuple(Δ)
```

Comment:
You can just refer to the function as `pairs_namedtuple`, so you don't need the assignment to `fun`.